### ISSUE TITLE
chart doesn't show

### ISSUE THREAD

**Issue by adriangalilea** [miguelmota/cointop#326]
macOS
brew install

Tested in 2 terminals warp and kitty
<img width="1431" alt="image" src="https://github.com/cointop-sh/cointop/assets/90320947/aff2007f-024c-495f-9bca-f53b69ad1ce2">

**Comment by adriangalilea** [comment]
Edit: it randomly started working now.

**Comment by o6uoq** [comment]
I have the same issue.

**Comment by zedxot** [comment]
> Edit: it randomly started working now.

I don't think it's random. I had the same problem before, and after a fresh OS install, now. Do you remember changing anything? Maybe compositor config or font.

**Comment by adriangalilea** [comment]
> I don't think it's random. I had the same problem before, and after a fresh OS install, now. Do you remember changing anything? Maybe compositor config or font.

Nope, I didn't change anything. I think it simply wasn't loading for some connection issue but there's no tell in the UI besides the missing chart.
